docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -116,6 +116,8 @@ This section describes optional configuration options, particularly for ensuring
 * `sip_tcp_send_timeout` - the time in milliseconds to wait for sent data to be acknowledgered at the TCP level on a SIP TCP connection (defaults to 2000 milliseconds).
 * `session_continued_timeout_ms` - if an Application Server with default handling of 'continue session' is unresponsive, this is the time that Sprout will wait (in milliseconds) before bypassing the AS and moving onto the next AS in the chain (defaults to 2000 milliseconds).
 * `session_terminated_timeout_ms` - if an Application Server with default handling of 'terminate session' is unresponsive, this is the time that Sprout will wait (in milliseconds) before terminating the session (defaults to 4000 milliseconds).
+* `pbxes` - a comma separated list of IP address that Bono considers to be PBXs that are incapable of registering. Non-REGISTER requests from these addresses are passed upstream to Sprout with a `Proxy-Authorization` header that causes Sprout to challenge it. Bono also permits requests to these addresses from the core to pass through it.
```

Comment:
I don't mind whether the plural is PBXs, PBXes or PBXen, but we should be consistent.